.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        # TODO: We need to re-introduce Node 6 and 7 once we have a way to run tests on them.
```

Comment:
@JoshuaKGoldberg probably this is not relevant once the ESM migration is ready :wink: